networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/GossipTopics.java

```diff
@@ -73,11 +74,14 @@ public static Set<String> getAllTopics(
     for (int i = 0; i < NetworkConstants.SYNC_COMMITTEE_SUBNET_COUNT; i++) {
       topics.add(getSyncCommitteeSubnetTopic(forkDigest, i, gossipEncoding));
     }
-    if (spec.getNetworkingConfigDeneb().isPresent()) {
-      for (int i = 0; i < spec.getNetworkingConfigDeneb().get().getBlobSidecarSubnetCount(); i++) {
-        topics.add(getBlobSidecarSubnetTopic(forkDigest, i, gossipEncoding));
-      }
-    }
+    spec.getNetworkingConfigDeneb()
```

Comment:
It definitely looks more readable without functional style. We've reverted it. Maybe keep it untouched?

Reply:
oh yeah can revert that, was trying to see how it looks and forgot to revert it.